src/debugger/adapter/DebugAdapter.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import { DebugSession, LoggingDebugSession, InitializedEvent } from "vscode-debugadapter";
+import * as DebugProtocol from "vscode-debugprotocol";
+
+class PrepackDebugSession extends LoggingDebugSession {
+  /**
+   * Creates a new debug adapter that is used for one debug session.
+   * We configure the default implementation of a debug adapter here.
+   */
+  constructor() {
+    super("prepack");
+    this.setDebuggerLinesStartAt1(true);
```

Comment:
Do both VSCode and Nuclide adapter expect line number starting at one instead of zero? If so, please add an explicit comment for this.

Reply:
From what I've seen, both can accept line number starting at one. I will change it if that's not the case when we try to connect to an IDE.